streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBWindowStore.java

```diff
@@ -52,10 +52,12 @@
     @Override
     public void init(final StateStoreContext context, final StateStore root) {
         super.init(context, root);
+        this.position = super.getPosition();
```

Comment:
Likewise here, it seems like we should be checkpointing this position?

Reply:
Same, should we checkpoint stores that implement WrappedStateStore?